app/components/search/Sidebar/Sidebar.tsx

```diff
@@ -28,16 +30,42 @@ const Sidebar = ({
     setSearchRadius(evt.target.value);
   };
 
+  const eligibilityNames = useMemo(() => {
+    // This var holds a set of eligibility names pertaining to a given whitelabeled category. This
+    // set can be used to filter the collection of eligibilities returned by Algolia in cases where
+    // we only want to display a specific list of eligibilities for said whitelabeled category.
+
+    // Currently, this is only set up to work with UCSF, but can be expanded upon in the future.
```

Comment:
I was kind of cheating here since any non-UCSF slug will not index the `ucsfEligibiltiesMap`; therefore this logic only works for UCSF routes implicitly. I get that it is kind of confusing/ kicks the ball forward for if/when we do want to set up limited eligibility filters for other whitelabels.

I had thought of three options for solving this in a more abstract way:

1. Add an ID to each whitelabel config and check that the ID is `UCSF` or whatever when defining the `eligibilityNames` var
2. Create a universal file which lists predefined eligibility sets for the different whitelabels and use a key property defined in the whitelabel config to index the desired set (e.g., `ucsfEligibilities`, `sffamilyEligibilities`, etc...)
3. Create separate files for each whitelabel eligibility set and create a map that uses a whitelabel config value (probably also `ucsfEligibilities`, `sffamilyEligibilities`, etc...) that points to the actual imported file in the `Sidebar`

I think my preferred method would be #2, which may also involve importing the existing `ucsfEligibilityMap` into the universal file and slicing up its data to better match what we're trying to do here (rather than listing a set of eligibilities in both the `ucsfEligibilityMap` and this file and having to keep them aligned )...

Alternatively, I refactored my code to make what I'm doing clearer, though I'd understand if you think going with one of the other above options is better

  const eligibilityNames = useMemo(() => {
    // This var holds a set of ........
    ......

    // This only accepts UCSF specific slugs because UCSF is currently the only whitelabel that
    // has a defined and limited set of eligibility filters.
    const resourceEligibilityGroups = ucsfEligibilityMap[categorySlug];
    if (!resourceEligibilityGroups) {
      return [];
    }

    const flatEligibilities = resourceEligibilityGroups.flatMap(group => (group.eligibilities));
    return flatEligibilities.map(eligibility => eligibility.name);
  }, []);

Reply:
Thanks for the explanation, and sorry for not reading this until now. I think kicking the can down the road probably makes sense for now, especially if we're going to have Product-level discussions about simplifying/unifying the eligibilities and categories.